<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GrabConfigWidget</class>
 <widget class="QWidget" name="GrabConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>152</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">GrabConfigWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_Red">
     <property name="text">
      <string>Red</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_Green">
     <property name="text">
      <string>Green</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_Blue">
     <property name="text">
      <string>Blue</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox_IsAreaEnabled">
     <property name="text">
      <string>Area enabled</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_Percent_1">
     <property name="text">
      <string notr="true">%</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_Percent_2">
     <property name="text">
      <string notr="true">%</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_Percent_3">
     <property name="text">
      <string notr="true">%</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBox_Red">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="spinBox_Green">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="spinBox_Blue">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>spinBox_Red</tabstop>
  <tabstop>spinBox_Green</tabstop>
  <tabstop>spinBox_Blue</tabstop>
  <tabstop>checkBox_IsAreaEnabled</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
